### PR TITLE
W-16161783 search files google

### PR DESCRIPTION
**Pull request by Cristian-Venticinque**

### CODE REVIEW

File: rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc [hunk 1]
```diff
@@ -0,0 +1,57 @@
+= Search in Google Drive
+
+Search for files or folders in your Google Drive, including your shared drives.
+
+You can add variables, filters, and operations to your search query. This action step returns the number of matches found, the file or folder IDs, and the names of the files or folders found.
```
Comment (ahelmbold) [inline rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc]: Change "operations" to "operators".

File: rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+* *Common* 
++
+See xref:common-properties.adoc[Common Properties]
++
+** *Include all files from shared drives* 
++
+Select this option to search files and folders in your shared drives in addition to My Drive. 
+** *Search Query*
++
+String containing the search query. Use the Wizard for assistance while building your query.
```
Comment (ahelmbold) [inline rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc:20]: Maybe add that that, additional to the provided ones, all filters are supported, and users can look them up here https://developers.google.com/drive/api/guides/ref-search-terms

File: rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
++
+Returns `true` if at least one result was found. 
+
+== Wizard 
+
+include::partial$oauth-authentication-wizard.adoc[tag=authentication-info]
+
+Select either *Files* or *Folders*, depending on the type of search to perform. 
+
+Specify your search terms in the *Search Query* textbox. You can insert filters and operators into your search query by selecting them from the *Insert filters* and *Insert operators* drop-down lists.
```
Comment (ahelmbold) [inline rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc]: Add that, additional to the provided ones via the drop-down lists, all filters and operators are supported as of https://developers.google.com/drive/api/guides/ref-search-terms

**Review by ahelmbold — CHANGES_REQUESTED** [review]
See comments

File: rpa-builder/6.7/modules/ROOT/pages/toolbox-google-drive-operations.adoc [hunk 4]
```diff
@@ -1,12 +1,15 @@
-= Google Drive Operations
+= Google Workspace
```
Comment (hannanelson) [inline rpa-builder/6.7/modules/ROOT/pages/toolbox-google-drive-operations.adoc:1]:
```suggestion
= Google Workspace Operations
```

File: rpa-builder/6.7/modules/ROOT/pages/toolbox-google-drive-operations.adoc [hunk 5]
```diff
@@ -1,12 +1,15 @@
...
 
-Google Drive operations enable you to access your Google Drive and perform operations such as downloading or exporting, uploading, or searching for files.
+Google Workspace operations enable you to access your Google Drive and perform operations such as downloading or exporting, uploading, or searching for files or folders.
```
Comment (hannanelson) [inline rpa-builder/6.7/modules/ROOT/pages/toolbox-google-drive-operations.adoc]: Flip to put user task first.

File: rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc [hunk 6]
```diff
@@ -0,0 +1,59 @@
+= Search in Google Drive
+
+Search for files or folders in your Google Drive, including your shared drives.
+
+You can add variables, filters, and operators to your search query. This action step returns the number of matches found, the file or folder IDs, and the names of the files or folders found.
```
Comment (hannanelson) [inline rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc]: Should these two sentences be flipped?

File: rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+You can add variables, filters, and operators to your search query. This action step returns the number of matches found, the file or folder IDs, and the names of the files or folders found. 
+
+== Properties 
+
+include::partial$oauth-authentication-settings.adoc[]
+
+* *Common* 
++
+See xref:common-properties.adoc[Common Properties]
```
Comment (hannanelson) [inline rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc]:
```suggestion
See xref:common-properties.adoc[Common Properties].
```

Comment (hannanelson) [reply rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc]: Hard-coded link text?

File: rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
++
+** *Include all files from shared drives* 
++
+Select this option to search files and folders in your shared drives in addition to My Drive. 
+** *Search Query*
++
+String containing the search query. 
++
+Use the Wizard for assistance while building your query. You can use any of Google's supported https://developers.google.com/drive/api/guides/ref-search-terms[search query terms and operators].
```
Comment (hannanelson) [inline rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc]: Should Wizard be lc?
Avoid using possessive with product names. ("...use any of the supported Google search..."

File: rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+
+include::partial$oauth-authentication-settings.adoc[]
+
+* *Common* 
++
+See xref:common-properties.adoc[Common Properties]
++
+** *Include all files from shared drives* 
++
+Select this option to search files and folders in your shared drives in addition to My Drive.
```
Comment (hannanelson) [inline rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc]: Make descriptions parallel.

"Option to search..."

File: rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+== Inbound Variables 
+
+* *{A} replaced by* ... *{Z} replaced by*
+include::partial$oauth-inbound-variables.adoc[]
+
+== Outbound Variables 
+
+* *Amount found matches*
++
+Number of files or folders found.
```
Comment (hannanelson) [inline rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc]: Global: All sentence fragments = no period.

File: rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
++
+Returns `true` if at least one result was found. 
+
+== Wizard 
+
+include::partial$oauth-authentication-wizard.adoc[tag=authentication-info]
+
+Select either *Files* or *Folders*, depending on the type of search to perform. 
+
+Specify your search terms in the *Search Query* textbox. The *Insert filters* and *Insert operators* drop-down lists contain the most common filters and operators. However, you can use any of Google's supported https://developers.google.com/drive/api/guides/ref-search-terms[search query terms and operators].
```
Comment (hannanelson) [inline rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc]:
```suggestion
Enter your search terms in the *Search Query* textbox. The *Insert filters* and *Insert operators* dropdown lists contain the most common filters and operators. However, you can use any supported Google https://developers.google.com/drive/api/guides/ref-search-terms[search query terms and operators].
```

Comment (hannanelson) [reply rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc]: Dropdown lists or menus?

File: rpa-builder/6.7/modules/ROOT/pages/toolbox-google-drive-operations.adoc (hunk 4 above)
Comment (Cristian-Venticinque) [reply rpa-builder/6.7/modules/ROOT/pages/toolbox-google-drive-operations.adoc:1]: This matches the name of the container as it shows in the UI

File: rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc (hunk 6 above)
Comment (Cristian-Venticinque) [reply rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc]: The first sentence is what the action step does, the second one shows optional configuration parameters. Maybe the last sentence can be moved after the first one as it explains what's the result of executing this action step.

**Pull request merged**